coalib/output/ConsoleInteraction.py

```diff
@@ -749,6 +749,8 @@ def show_bear(bear,
                          '  ',
                          'This bear cannot fix issues or does not provide '
                          'information about what categories it can fix.')
+        console_printer.print(
+            '  Path:\n' + '   ' + bear.file_path + '\n\n')
```

Comment:
Use `repr(bear.file_path)` if spaces exist so that you don't need to worry about single quotes, double quotes, ...